sensor_msgs/PointCloud2 point_cloud_map
---
bool success